src/molecule.py

```diff
@@ -2987,11 +2947,7 @@ def read_inpcrd(self, fnm, **kwargs):
             if ln == 0:
                 comms = [line]
             elif ln == 1:
-                # Although is isn't exactly up to spec, 
```

Comment:
The lines highlighted in red are newer, so the changes here need to be reverted, perhaps by merging the master branch into this branch.